package.json

```diff
@@ -24,7 +24,7 @@
     "lerna": "^3.22.1",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.6.2",
-    "typescript": "~4.6.3"
+    "typescript": "beta"
```

Comment:
I'd prefer a specific version (`4.7.0-beta`) instead of a generic `beta` label

Reply:
This is a draft PR and in the description I said the proposed change would be 4.7, i.e. `~4.7.0` here.

Are you saying you think `4.7.0-beta` is okay to merge to master?